### PR TITLE
Json export for aggregated proof

### PR DESCRIPTION
**Pull request by Iluvmagick**

### CODE REVIEW

File: transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+                            inner_after_first = true;
+                        }
+                        out << "\t\t\t]}" << std::endl;
+                        map_after_first = true;
+                    }
+                    out << "\t\t]}" << std::endl;
+                    after_first = true;
+                }
+                out << "\t\t]}," << std::endl;
+                // and now serialize one of the merkle proofs
```
Comment (ETatuzova) [inline transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp:270]: Merkle proof consists of pairs <hash, position>, where position is {0,1}
We need hashes for all initial proofs.

Positions should be stored only once.

**Review by ETatuzova — CHANGES_REQUESTED** [review]

File: transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                const auto &aggregated_proof = proof.aggregated_proof;
+                out << "\t{\"struct\":[" << std::endl;
+                // single fri proof checking that F(x) is low degree
+
+                // basic_fri::round_proofs_batch_type fri_round_proof
+                const auto &fri_round_proof = aggregated_proof.fri_proof;
+                // which is in essence std::vector<std::vector<round_proof_type>> round_proofs;
+                out << "\t\t{\"array\":[" << std::endl;
+                bool after_first = false;
+                for (const auto &outer_proof_vector : fri_round_proof.fri_round_proof.round_proofs) {
```
Comment (ETatuzova) [inline transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp:161]: We also need round proof merkle paths. Only hashes. Not positions

File: transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp (hunk 1 above)
Comment (Iluvmagick) [reply transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp]: Should be fixed now.

File: transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp (hunk 2 above)
Comment (Iluvmagick) [reply transpiler/include/nil/blueprint/transpiler/aggregated_verifier_generator.hpp:161]: Should be fixed now.

**Review by ETatuzova — APPROVED** [review]

**Pull request merged**